cilium/cmd/helpers.go

```diff
@@ -385,3 +390,35 @@ func dumpConfig(Opts map[string]string) {
 		}
 	}
 }
+
+// getIpv6EnableStatus api returns the EnableIPv6 status
+// by consulting the cilium-agent otherwise reads from the
+// runtime system config
+func getIpv6EnableStatus() bool {
+	params := daemon.NewGetHealthzParamsWithTimeout(5 * time.Second)
+	brief := true
+	params.SetBrief(&brief)
+	// If cilium-agent is running get the ipv6 enable status
+	if _, err := client.Daemon.GetHealthz(params); err == nil {
+		if resp, err := client.ConfigGet(); err == nil {
+			if resp.Status != nil {
+				return resp.Status.Addressing.IPV6 != nil && resp.Status.Addressing.IPV6.Enabled == true
+			}
+		}
+	} else { // else read the EnableIPv6 status from the file-system
+		agentConfigFile := filepath.Join(defaults.RuntimePath, defaults.StateDir,
+			"agent-runtime-config.json")
+		var agentConfig interface{}
+
+		if _, err := os.Stat(agentConfigFile); !os.IsNotExist(err) {
+			if byteValue, err := ioutil.ReadFile(agentConfigFile); err == nil {
+				if err = json.Unmarshal(byteValue, &agentConfig); err == nil {
+					ipV6Status := agentConfig.(interface{}).(map[string]interface{})["EnableIPv6"]
+					return ipV6Status.(interface{}).(bool)
+				}
+			}
+		}
+	}
+	// returning the EnableIPv6 default status
+	return true
```

Comment:
This should probably be `defaults.EnableIPv6`.

Reply:
Done.